pkg/types/clusterconfig/cluster_config_gcp.go

```diff
@@ -198,6 +212,13 @@ var UserGCPValidation = &cr.StructValidation{
 				Validator: validateImageVersion,
 			},
 		},
+		{
+			StructField: "ImageClusterAutoscaler",
```

Comment:
The cluster-autoscaler image should be moved from `non_dev_images_aws` to `non_dev_images_cluster` in `images.sh`